aws_lambda_scripts/main.py

```diff
@@ -51,20 +51,20 @@ def handler(event, context):
     print("Secret retrieved")
 
     # Get updated copilot usage data from GitHub API
-    access_token = api_interface.get_access_token(org, secret, client_id)
+    access_token = github_api_toolkit.get_token_as_installation(org, secret, client_id)
 
     if type(access_token) == str:
         return(f"Error getting access token: {access_token}")
     else:
         print("Access token retrieved")
 
     # Create an instance of the api_controller class
-    gh = api_interface.api_controller(access_token[0])
+    gh = github_api_toolkit.github_interface(access_token[0])
 
     print("API Controller created")
```

Comment:
I'll open a separate jira ticket to clean up logging.  Ideally we would use the python logging module and set a commen format for our logging, this will enable easier debugging when logs are streamed to cloudwatch. It doesn't need to be fixed as part of this PR.

Reply:
Jira ticket added, will complete in next sprint